src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailedPackageMetadata.cs

```diff
@@ -41,6 +40,26 @@ public DetailedPackageMetadata(IPackageSearchMetadata serverData, long? download
             PrefixReserved = serverData.PrefixReserved;
             LicenseMetadata = serverData.LicenseMetadata;
             _localMetadata = serverData as LocalPackageSearchMetadata;
+
+            // Determine the package detauls URL and text.
+            PackageDetailsUrl = null;
+            PackageDetailsText = null;
+            if (serverData.PackageDetailsUrl != null
+                && serverData.PackageDetailsUrl.IsAbsoluteUri
+                && serverData.PackageDetailsUrl.Host != null)
+            {
+                PackageDetailsUrl = serverData.PackageDetailsUrl;
+                PackageDetailsText = serverData.PackageDetailsUrl.Host;
+
+                // Special case the display of "www.nuget.org" to "nuget.org" since we know "nuget.org"
+                // redirects to "www.nuget.org". We chose to show the full host (including subdomain) because
+                // this is significant for some feeds. For example, "{name}.myget.org" can be a different
+                // managing organization than "www.myget.org" which can be a significant difference in intent.
+                if (string.Equals("www.nuget.org", PackageDetailsText, StringComparison.OrdinalIgnoreCase))
```

Comment:
🤢

Reply:
Yeah, I wasn't too thrilled with this either. Alternatives? 

I considered:

1. Just showing "www.nuget.org" - this is wrong because our "brand name" is nuget.org pretty much universally in docs.
1. Enhancing the protocol to enable a display name - this would allow for all kind of weirdness
1. Enhancing the protocol to have a "hide www" bool - it's not even clear where we would put this information
1. Put "nuget.org" in the template - this would have a redirect every time and does have failover. Basically nuget.org only has an A record that points to USNC. "www.nuget.org" is what has the cross-regional failover.

Given these things, I thought the simplest approach was the best. Of these options though, I think # 1 is the best...

Reply:
Let's just hide "www" on all domains. This is what mobile Chrome does on iPhone, apparently (spoke to @anangaur about this).

Reply:
I'm ok with that.

Reply:
Done, with unit tests.